modules/persistent/arrow-persistent-data-structures/build.gradle

```diff
@@ -0,0 +1,16 @@
+apply plugin: 'kotlin-kapt'
+
+dependencies {
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
+    compile project(':arrow-annotations')
+    compile project(':arrow-core')
+    kapt project(':arrow-meta')
+    kaptTest project(':arrow-meta')
+    compileOnly project(':arrow-meta')
```

Comment:
```suggestion
```

This shouldn't be necessary, it doesn't affect the binary but it clouds your IDEA suggestions.